### PR TITLE
Install Java SDK in Dockerfile

### PR DESCRIPTION
**Pull request by wangkuiyi**
so to enable .pre-commit-config.yaml to check Java code in CI.

### CODE REVIEW

File: scripts/image_build.sh [hunk 1]
```diff
@@ -39,22 +39,27 @@ pre-commit \
...
 go get github.com/golang/protobuf/protoc-gen-go
 mv $GOPATH/bin/protoc-gen-go /usr/local/bin/
 go get golang.org/x/lint/golint
 mv $GOPATH/bin/golint /usr/local/bin
 
 # 3. Install protobuf compiler
-wget --quiet https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip
-unzip -qq protoc-3.6.1-linux-x86_64.zip -d /usr/local
-rm protoc-3.6.1-linux-x86_64.zip
+wget -q https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip
```
Comment (typhoonzero) [inline scripts/image_build.sh:55]: Just curious, is gRPC java protobuf-compiler plugin requires protoc `v3.7.1`?

**Review by typhoonzero — APPROVED** [review]
LGTM

File: scripts/image_build.sh (hunk 1 above)
Comment (wangkuiyi) [reply scripts/image_build.sh:55]: No. It works with 3.6 too. Just trying to keep everything up-to-date.

Comment (typhoonzero) [reply scripts/image_build.sh:55]: Cool.

**Pull request merged**